analysis_configs/sequences.py

```diff
@@ -88,6 +88,15 @@ def add_good_ak8_jet_branch(events):
     return events
 
 
+def add_good_ak4_jet_branch(events):
+    events["Jets"] = ak.with_field(
+        events["Jets"],
+        obj.is_good_ak4_jet(events["Jets"]),
+        "isGood",
+    )
+    return events
+
```

Comment:
If we are adding an "isGood" branch for ak4 jets here, would it make sense to also add "isVetoLepton" branch?

Reply:
But then that adds many layers of things that are the same... The good ak8 jet branch needs to be added to the output files because we use that afterwards for training. And we veto events with veto leptons, so that branch would be always filled with False! It does not bring any info!
I thought of adding the good ak4 jets branch because it can be useful. Like making the map that you do, after applying the phi spike filter, to check it looks good.